covalent/_file_transfer/enums.py

```diff
@@ -29,6 +29,7 @@ class Order(str, enum.Enum):
 class FileSchemes(str, enum.Enum):
     File = "file"
     S3 = "s3"
+    Blob = "blob"
```

Comment:
Is the Azure blob URI scheme "blob" or ["abfss"](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction-abfs-uri)?

Reply:
It looks like it uses [https](https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata) actually; we would need a separate interface for ADLS.  We can either use `blob` to distinguish these paths from generic `https` or stick with https and discourage scheme inference in the case of Azure blob storage.

Reply:
@santoshkumarradha

Reply:
current method filters by https and also checks for `blob.core.windows.net` in the `host` portion of the URI in order to determine a Blob transfer strategy.